EventKit/eventkit-cloud
# Add most download sort

## Description of Improvement

Adds sort options to the UI, for sorting by most popular download and most recently downloaded. 

## How to test

To test, create a couple of jobs featuring a couple of different products.  Then download data and then create a new job ensuring that the correct products are represented via the sort options.  The actual api response should be verified to ensure that the expected products have the correct label. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
